src/js/components/Image/stories/Simple.js

```diff
@@ -5,7 +5,7 @@ import { Image } from 'grommet';
 export const Simple = () => (
   // Uncomment <Grommet> lines when using outside of storybook
   // <Grommet theme={...}>
-  <Image src="//v2.grommet.io/assets/IMG_4245.jpg" />
+  <Image src="//v2.grommet.io/assets/IMG_4245.jpg" alt="image" />
```

Comment:
Can this be changed so that the alt text has a more meaningful description for screenreader users?

Reply:
update the alt text